keras/engine/training.py

```diff
@@ -926,6 +927,13 @@ def fit(self,
             validation_steps: Only relevant if `steps_per_epoch`
                 is specified. Total number of steps (batches of samples)
                 to validate before stopping.
+            validation_freq: Only relevant if validation data is provided. Integer
+                or `collections.Container` instance (e.g. list, tuple, etc.). If an
+                integer, specifies how many training epochs to run before a new
+                validation run is performed, e.g. `validation_freq=2` runs
+                validation every 2 epochs. If a Container, specifies the epochs on
```

Comment:
I'm afraid "Container" is not a term that can be readily understood by most users. Please say "list, tuple, or set"

Reply:
Thanks for the review! Done